src/SamplesApp/UITests.Shared/Windows_Media/MediaPlayerTests.xaml

```diff
@@ -0,0 +1,23 @@
+﻿<Page
+    x:Class="UITests.Windows_Media.MediaPlayerTests"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:UITests.Windows_Media"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    xmlns:valueConverters="using:UITests.ValueConverters"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+
+	<UserControl.Resources>
+		<valueConverters:SoundSourceConverter x:Key="soundSourceConverter" />
+	</UserControl.Resources>
+
+	<StackPanel>
+		<MediaPlayerElement x:Name="playbaby"
+		                    Source="{x:Bind Model.SoundToPlay, Mode=OneWay, Converter={StaticResource soundSourceConverter}}"
```

Comment:
You should be able to use x:Bind this way:

```suggestion
		                    Source="{x:Bind ToMediaPlaybackItem(Model.SoundToPlay)}"
```

And have a method here, directly in the `MediaPlayerTests` page, or in a static class:

```csharp
private static MediaPlaybackItem ToMediaPlaybackItem(object value)
{
		if(value is string uriString && Uri.TryCreate(uriString, out var uri))
		{
				var source = MediaSource.CreateFromUri(uri);
				return new MediaPlaybackItem(source);
		}

		return null;
}
```

There's no convert back in this case, but it can also be added as a method if needed.

Reply:
whoa first time using `x:Bind`, didn't realize it could do that!